quickstart/README.md

```diff
@@ -51,40 +51,39 @@ The express install script will do quite a few things to get you bootstrapped.
     1. create a full suite of PKI
     1. create a config file for the controller
     1. create a config file for an edge router
-1. download the latest distribution of ziti from github.com/openziti/ziti/releases
+1. download the latest distribution of Ziti from github.com/openziti/ziti/releases
 1. unzip the distribution
-1. start the `ziti-controller` and `ziti-router` executables
-1. the `ziti-controller` should now be exposed on https://$(hostname):1280
+1. start the controller and router in the background
+1. expose the controller on https://$(hostname):1280
 
 ## Docker - Compose
 
-The [docker-compose](https://docs.docker.com/compose/) based example will create numerous `ziti-router`s 
+The [docker-compose](https://docs.docker.com/compose/) based example will create numerous `ziti-router`s
```

Comment:
You plan to find all `ziti-router` too? On another pr maybe?

Reply:
I could do it in this one, I guess. :slightly_smiling_face:

Reply:
De-hyphenated ziti-router where practical in https://github.com/openziti/ziti/pull/1968